perfmetrics/scripts/fio/fio_metrics.py

```diff
@@ -152,75 +278,37 @@ def _load_file_dict(self, filepath) -> Dict[str, Any]:
       raise NoValuesError(f'JSON file {filepath} returned empty object')
     return fio_out
 
-  def _extract_metrics(self, fio_out) -> List[Dict[str, Any]]:
-    """Extracts and returns required metrics from fio output dict.
-
-      The extracted metrics are stored in a list. Each entry in the list is a
-      dictionary. Each dictionary stores the following fio metrics related
-      to a particualar job:
-        jobname, filesize, number of threads, IOPS, Bandwidth and latency (min,
-        max and mean)
+  def _get_start_end_times(self, out_json, job_params) -> List[Tuple[int]]:
+    """Returns start and end times of each job as a list.
 
     Args:
-      fio_out: JSON object representing the fio output
+      out_json : FIO json output
+      job_params: List of dicts, Parameters of all jobs
 
     Returns:
-      List of dicts, contains list of jobs and required metrics for each job
-      Example return value:
-        [{'jobname': '1_thread', 'filesize': 50000, 'num_threads': 40, 'rw': 'read'
-        'start_time': 1653027084, 'end_time': 1653027155, 'iops': 95.26093,
-        'bw_bytes': 99888128, 'lat_s': {'min': 0.35337776000000004, 'max':
-        1.6975198690000002, 'mean': 0.41775487677469203, 'lat_20_perc':
-        0.37958451200000004, 'lat_50_perc': 0.38797312, 'lat_90_perc':
-        0.49283072000000006, 'lat_95_perc': 0.526385152}, 'io_bytes':
-        6040846336}]
-
-    Raises:
-      KeyError: Key is missing in the json output
-      NoValuesError: Data not present in json object
+      List of start and end time tuples, one tuple for each job
+      Ex: [(1653027014, 1653027084), (1653027084, 1653027155)]
 
     """
-
-    if not fio_out:
-      raise NoValuesError('No data in json object')
-
-    global_filesize = ''
+    rw_rev_list = [job_param[RW] for job_param in reversed(job_params)]
```

Comment:
why do you need reversed. can you add a comment here.

Reply:
done